core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goal-configuration-url-parameter-component/dot-experiments-goal-configuration-url-parameter-component.component.html

```diff
@@ -7,14 +7,15 @@ <h3>{{ 'experiments.goal.reach_page.form.conditions.label' | dm }}</h3>
             formArrayName="conditions"
         >
             <div class="condition flex" [formGroupName]="i">
-                <div class="field flex-grow-1 flex flex-column w-15rem">
+                <div class="field flex-grow-1 flex flex-column w-15rem" formGroupName="value">
```

Comment:
you mean `formGroupName="name"` ?

Reply:
No, in `URL_PARAMETER` value is a FormGroup, like:
```
conditions: [.   <- formArray iterated
                    {
                        parameter: GOAL_PARAMETERS.QUERY_PARAM,
                        operator: GOAL_OPERATORS.EQUALS,
                        value: { <-- FormGroup
                            name: 'test',
                            value: 'value'
                        }
                    }
                ]
```